### PR TITLE
Update Supplier.php

### PR DESCRIPTION
**Pull request by switch-cheehau**
I think <code>default</code> is better.

1. Not everyone fill in full address for supplier.
2. the comma will still display if one of the field left empty intentionally.
3. <code>default</code> supplier address will not have this issue.
4. I'm not sure original purpose of the <code>short</code> supplier address

Example: leaving company field empty purposely, as supplier was defined as outlets within same company.
image

### CODE REVIEW

**Comment by aimeos** [comment]
The alternative to commas as separators are newlines. If there's more than one subseqently, they can be trimmed like here:
https://github.com/aimeos/ai-client-html/blob/master/client/html/src/Client/Html/Checkout/Standard/Address/Billing/Standard.php#L563

**Comment by switch-cheehau** [comment]
> The alternative to commas as separators are newlines. If there's more than one subseqently, they can be trimmed like here:
> https://github.com/aimeos/ai-client-html/blob/master/client/html/src/Client/Html/Checkout/Standard/Address/Billing/Standard.php#L563

Yup understand that. the example you provided is for billing address.
As for supplier address used in "Click & Collect", if we can make the address format configurable between commas and newlines, i think it will be great.

File: lib/mshoplib/src/MShop/Service/Provider/Decorator/Supplier.php [hunk 1]
```diff
@@ -179,7 +179,7 @@ public function setConfigFE( \Aimeos\MShop\Order\Item\Base\Service\Iface $orderS
 		if( ( $code = $attributes['supplier.code'] ) != '' )
 		{
 			// add short address as attribute for summary page / customer email
-			$attributes['supplier.address'] = $this->feConfig['supplier.code']['short'][$code];
+			$attributes['supplier.address'] = $this->feConfig['supplier.code']['default'][$code];
```
Comment (switch-cheehau) [inline lib/mshoplib/src/MShop/Service/Provider/Decorator/Supplier.php:182]: I think with this change, it allow us to choose the address format, but i don't know how to call config from this file.

$attributes['supplier.address'] = $this->feConfig['supplier.code'][  <code>$this->config('client/format','default')</code>  ][$code];

Comment (aimeos) [reply lib/mshoplib/src/MShop/Service/Provider/Decorator/Supplier.php:182]: `$this->getConfigValue( 'supplier.addressformat' )` will do the job if you add it to the `$beConfig`

**Comment by switch-cheehau** [comment]
will close this PR due to messy commits history

**Pull request closed without merging**